appointments/cmd/main/handlers/modify_rdv_doctor.go

```diff
@@ -0,0 +1,129 @@
+package handlers
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/edgar-care/appointments/cmd/main/lib"
+	"github.com/edgar-care/appointments/cmd/main/services"
+	"github.com/go-chi/chi/v5"
+)
+
+func UpdateDoctorAppointment(w http.ResponseWriter, req *http.Request) {
+	doctorID := lib.AuthMiddlewareDoctor(w, req)
+	if doctorID == "" {
+		lib.WriteResponse(w, map[string]string{
+			"message": "Not authenticated",
+		}, 401)
+		return
+	}
+
+	appointmentID := chi.URLParam(req, "id")
+
+	if appointmentID == "" {
+		lib.WriteResponse(w, map[string]string{
+			"message": "ID is not defined",
+		}, 400)
+		return
+	}
+
+	var new_appointment services.RdvInput
+	err := json.NewDecoder(req.Body).Decode(&new_appointment)
+	lib.CheckError(err)
+
+	appointment, err := services.GetRdvById(new_appointment.Id)
+	lib.CheckError(err)
+	if err != nil {
+		lib.WriteResponse(w, map[string]string{
+			"message": "Id not correspond to an appointment",
```

Comment:
"Id **does** not correspond to an appointment" (y'en a plusieurs dans le fichier)